commands/commands.go

```diff
@@ -99,6 +101,19 @@ func (c *Command) Run(pctx context.Context, bus *events.EventBus) {
 			bus.Publish(events.Event{events.Error, err.Error()})
 			return
 		}
+
+		// if we're able to, log the PID of our Command's exec process through
+		// our logger fields
+		if c.Cmd != nil && c.Cmd.Process != nil {
+			pid := c.Cmd.Process.Pid
+			if len(c.fields) > 0 {
+				c.fields["pid"] = pid
+			} else {
+				c.fields = log.Fields{"pid": pid}
+			}
+			c.logger = *log.WithFields(c.fields)
```

Comment:
Actually, we attach to the process stdout/stderr well before this part -- so does this work? We should definitely verify the pid field shows up in the `test_logging` integration test.